pro_1.py

```diff
@@ -0,0 +1,100 @@
+import json
+import Patterns
+import config_st
+import database_interaction
+import User
+import re 
+from re import findall
+database_file=open("database.txt","r+")
```

Comment:
I see two database files?! why do we need the DB here? dont we have database_interaction!

Reply:
to compare the input email with store email in the database file